interpreter/cling/CMakeLists.txt

```diff
@@ -1,12 +1,4 @@
-if(WIN32)
-  # We need cmake to support exporting of symbols not only from libraries but
-  # from executables too. This way cling can find symbols from its own
-  # executable during runtime.
-  cmake_minimum_required(VERSION 3.6.2)
-else(WIN32)
-  # support of earlier cmake versions will be removed soon
-  cmake_minimum_required(VERSION 3.5)
-endif(WIN32)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
What is the default CMake version for the oldest distribution we support?

Reply:
It's 3.26.5 on `alma8` and `3.22.1` on Ubuntu 22.04, so we're safe there. Note that Ubuntu 20.04 has version 3.16.3, but that distribution will be EOL by the time ROOT 6.36 comes out